### ISSUE TITLE
Add support for v4 shims with v5

### ISSUE THREAD

**Issue by bugfolder** [backdrop-contrib/font_awesome#15]
FontAwesome v5 renamed some icons, which breaks some v4 icon names. But FontAwesome also [provides a shim file](https://fontawesome.com/v5.15/how-to-use/graphql-api/objects/shim) that restores compatiblity with v4 while allowing one to use all the goodness of v5. It would be nice to offer that as a option, which the attached PR does.

To see the difference, put this code in a page and compare the 3 configuration settings:

```
<p>
FontAwesome 4 icons (bandcamp, cart): <i class="fa fa-bandcamp"></i>, <i class="fa fa-shopping-cart"></i>
</p>

<p>
FontAwesome 5 icons (bandcamp, cart): <i class="fab fa-bandcamp"></i>, <i class="fas fa-shopping-cart"></i>
</p>
```

"v4" and "v5" don't display all of the icons, but the "v5 + shims" option will display both pairs of icons.

![fontawesome](https://user-images.githubusercontent.com/29214084/148318190-7d497843-87e3-4fdb-8ab1-f2ddf3854401.jpg)

**Comment by yorkshire-pudding** [comment]
Hi @bugfolder - Thank you for your contribution and sorry it has sat ignored for so long.
I've just taken over the maintainer-ship of this module so it doesn't get too neglected.  Your change looks mostly good, but needs a rebase (after #11) and it probably makes sense (if the shims work with that version) to update it to 5.15.4 or could match the versions with shims if that works; up to you.

Thanks

**Comment by bugfolder** [comment]
I rebased in a new PR.

I observe that now there are two v5 options: v5.2 and v5.15. That suggests that instead of making the shim option one of the radio buttons, the shim choice should be a separate checkbox that would apply to either of the v5 options (and/or any future v5 additions). Would you agree? If so, I'll update the PR.

**Comment by yorkshire-pudding** [comment]
@bugfolder - Yes - that makes sense.  Thank you

**Comment by bugfolder** [comment]
PR updated, ready for review.
